recipe/meta.yaml

```diff
@@ -1,71 +1,238 @@
+
+  - url: https://github.com/libarchive/libarchive/releases/download/v{{ libarchive_version }}/libarchive-{{ libarchive_version }}.tar.gz
+    sha256: ba7eb1781c9fbbae178c4c6bad1c6eb08edab9a1496c64833d1715d022b30e2e
+    folder: libarchive
+    patches:
+      - libarchive-patches/0001-Add-lib-to-CMAKE_FIND_LIBRARY_PREFIXES-for-lzma.patch
+      - libarchive-patches/0002-xar-fix-bad-seek-in-move_reading_point.patch
+      - libarchive-patches/0003-VC9-compatibility-test-for-BCryptDeriveKeyPBKDF2.patch
+      - libarchive-patches/0004-VC9-compatibility-define-BCRYPT_SUCCESS.patch
+      # This should not be applied in general. It exists so I can build it for
+      # debugging Visual Studio where warnings are errors for some reason.
+      # Hiding warnings is poor practice.
+      # - libarchive-patches/0005-VC9-compatibility-ignore-some-warnings.patch
+      - libarchive-patches/0006-VC-openssl-define-inline-as-__inline.patch
+      # These warnings are errors when compiling lz-4 support
+      - libarchive-patches/0007-CMake-VC9-compatibility-ignore-some-warnings.patch
+      - libarchive-patches/0008-VC9-compatibility-remove-C99.patch
+      - libarchive-patches/0009-CMake-Force-Multi-threaded-DLL-runtime.patch
+      # 3.4.0 added official support for symlinks on win
+      - libarchive-patches/0010-Use-CreateSymbolicLinkW-on-Windows-for-symlinks-WIP.patch
 
 build:
-  number: 2
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
-  entry_points:
-    - cph = conda_package_handling.cli:main
+  number: 4
   # HACK: case-sensitive mismatch in latest conda-build
   missing_dso_whitelist:
-    - "*/VCRUNTIME140.dll"   # [win]
-    - "MSVCR90.dll"   # [win]
+    - Library/bin/VCRUNTIME140.dll    # [win]
+    - "*/MSVCR90.dll"                 # [win]
+    - "MSVCR90.dll"                   # [win]
 
 requirements:
   build:
     - {{ compiler('c') }}
-  host:
-    - python
-    - pip
-    - cython
-    - _libarchive_static_for_cph
-    - msinttypes  # [win and vc<14]
-  run:
-    - futures   # [py27]
-    - python
-    - six
-    - tqdm
-
-test:
-  source_files:
-    - tests
-  requires:
-    - mock
-    - pytest
-    - pytest-cov
-    - pytest-mock
-  imports:
-    - conda_package_handling
-    - conda_package_handling.archive_utils_cy
-  commands:
-    # This test is skipped until Dec 2019 which is weird. Skip for now.
-    - pytest tests -k "not test_secure_refusal_to_extract_abs_paths"
+    - {{ compiler('cxx') }}
+    - autoconf    # [unix]
+    - automake    # [unix]
+    - libtool     # [unix]
+    - pkg-config  # [unix]
+    - cmake
+    - make        # [unix]
+    - m2-patch    # [win]
+    - patch       # [unix]
+    - jom         # [win]
+    - ninja       # [win]
+    - m2-gcc-libs  # [win]
+
+outputs:
+  # Recipe for a static libarchive with support for the features needed for
+  # conda-package-handling (bzip2, zlib, zstd).
+  # This is not a general purpose libarchive
+  - name: _libarchive_static_for_cph
```

Comment:
Why is this needed as an output?

Reply:
I don't know explicitly how to find the path to the global build when there are multiple outputs. Making an output was an easy hack.

Reply:
I mean, we can have only one output `conda-package-handling` and remove `_libarchive_static_for_cph` as an output.

Reply:
Yup I know. But how do I find the proper .a files from the new workdir? They are installed into $PREFIX when the global build runs. Then new environments are made for each output with a new PREFIX. IDK how to find the old PREFIX.

Reply:
> Then new environments are made for each output with a new PREFIX

Why do you need multiple outputs and new environments? Why can't this be one output and one environment?

Reply:
conda build makes them.

Reply:
There is a single build at the start that is executed by running `build.sh`. Then each output gets a new set of environments and is packaged. I am asking how to access files installed into the host env of the first global build.

Reply:
If we remove `outputs`, conda-build doesn't make them. It's just one `PREFIX`.

Reply:
one single build is hard because the build makes a bunch of files we don't need, like .a and .so's

Reply:
For example, we;d vendor .so's for zlib which would conflict with zlib itself

Reply:
conda build has too much magic - see the latest pr

Reply:
*commit